Mathlib/RingTheory/ZariskisMainTheorem.lean

```diff
@@ -0,0 +1,95 @@
+/-
+Copyright (c) 2025 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+module
+
+public import Mathlib.RingTheory.Ideal.Over
+public import Mathlib.RingTheory.IntegralClosure.IsIntegralClosure.Basic
+
+/-!
+# Algebraic Zariski's Main Theorem
+
+In this file we will prove (TODO @erdOne) the algebraic version of Zariski's main theorem
```

Comment:
If this is your intended strategy, could you state https://stacks.math.columbia.edu/tag/00Q9 in the docstring and relate this to your definition `ZariskisMainProperty`?

Reply:
I added something in the module docstring. Is this what you were thinking of?